### PR TITLE
Pull request to enable display of barometric pressure

### PR DESCRIPTION
**Pull request by stavefan**
Modification of code and readme that permits display of the barometric pressure.

### CODE REVIEW

**Comment by stavefan** [comment]
Attached is a screenshot

pressure

File: MMM-WeatherChart.js [hunk 1]
```diff
@@ -539,7 +581,9 @@ Module.register("MMM-WeatherChart", {
             y2_max =
                 Math.max(maxRain, maxSnow, this.config.rainMinHeight) *
                 (2 + iconTopMargin + iconBelowMargin + tempRainMargin),
-            y2_min = 0;
+            y2_min = 0,
+            y3_max = maxPressure + 20,
+            y3_min = minPressure - 20;
```
Comment (mtatsuma) [inline MMM-WeatherChart.js:586]: y3_max and y3_min should be carefully calculated to make an appropriate margin. If it's difficult, I'll try to fix after the PR is merged.

File: MMM-WeatherChart.js [hunk 2]
```diff
@@ -697,6 +751,32 @@ Module.register("MMM-WeatherChart", {
...
+		        },
+		        display: this.config.datalabelsDisplay,
+		        formatter: function (value) {
+		            let place = 10 ** self.config.datalabelsRoundDecimalPlace;
+		            let label = Math.round(value * place) / place;
+		            return label;
+		        },
+		    },
+		    data: pressures,
+		    yAxisID: "y3",
```
Comment (mtatsuma) [inline MMM-WeatherChart.js:775]: Hide the y3 axis at the far left side.

y1, y2 axis is hide by the below code.
https://github.com/mtatsuma/MMM-WeatherChart/blob/master/MMM-WeatherChart.js#L880-L884

**Review by mtatsuma — CHANGES_REQUESTED** [review]

File: MMM-WeatherChart.js [hunk 3]
```diff
@@ -784,7 +864,10 @@ Module.register("MMM-WeatherChart", {
             y2_max =
                 Math.max(maxRain, maxSnow, this.config.rainMinHeight) *
                 (2 + (iconTopMargin + iconBelowMargin + tempRainMargin) * 2),
-            y2_min = 0;
+            y2_min = 0,
+            yPressure_Min = this.getMin(pressures),
```
[the remark this answers is not in the document]
Comment (stavefan) [reply MMM-WeatherChart.js]: Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a

File: MMM-WeatherChart.js [hunk 4]
```diff
@@ -45,6 +46,7 @@ Module.register("MMM-WeatherChart", {
         colorMax: "rgba(255, 255, 255, 1)",
         colorRain: "rgba(255, 255, 255, 1)",
         colorSnow: "rgba(255, 255, 255, 1)",
+        colorPressure: "rgba(255, 255, 0, 0.8)",
```
[the remark this answers is not in the document]
Comment (stavefan) [reply MMM-WeatherChart.js]: Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a

File: MMM-WeatherChart.js [hunk 5]
```diff
@@ -452,6 +469,31 @@ Module.register("MMM-WeatherChart", {
             data: nightTemps,
             yAxisID: "y1",
         });
+        if (this.config.showPressure) {
+            datasets.push({
+                label: "Pressure",
+             borderColor: this.config.colorPressure,
+            pointBackgroundColor: this.config.colorPressure,
```
[the remark this answers is not in the document]
Comment (stavefan) [reply MMM-WeatherChart.js:477]: Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a

File: MMM-WeatherChart.js (hunk 1 above)
Comment (stavefan) [reply MMM-WeatherChart.js]: Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a

File: MMM-WeatherChart.js (hunk 2 above)
Comment (stavefan) [reply MMM-WeatherChart.js:779]: Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a

**Review by mtatsuma — APPROVED** [review]
Thank you for the modifitions.

**Comment by mtatsuma** [comment]
I'll create a new release after I test it on my environment.

**Pull request merged**